server/apiClient/SpoonacularClient.js

```diff
@@ -0,0 +1,18 @@
+import got from 'got'
+
+const spoonacularApiKey = process.env.SPOONTACULAR_API
+
+class SpoonacularClient {
+  static async getIngredient(ingredient) {
+    try {
+      const url = `https://api.spoonacular.com/food/ingredients/search?apiKey=${spoonacularApiKey}&query=${ingredient}`
+      const apiResponse = await got(url)
+      const responseBody = apiResponse.body
+      return responseBody
+    } catch (error) {
+      return { error: error.message }
+    }
+  }
+}
+
+export default SpoonacularClient
```

Comment:
not sure why this file is appearing twice... the latter is the up to date file

Reply:
had two apiClient folders with two client.js files in two different levels of my server directory -_-
deleted the above file